streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -389,6 +389,16 @@
      */
     <KR> KStream<KR, V> toStream(final KeyValueMapper<? super K, ? super V, ? extends KR> mapper);
 
+    /**
+     * Suppress some updates from this changelog stream, determined by the supplied {@link Suppress}.
+     *
+     * This controls what updates downstream table and stream operations will receive.
+     *
+     * @param suppress Configuration object determining what, if any, updates to suppress.
+     * @return A new KTable with the desired suppress characteristics.
+     */
+    KTable<K, V> suppress(final Suppress<K, V> suppress);
```

Comment:
yes, it should! Thanks for the catch and the reference.

Reply:
Actually, no it shouldn't ;)

`Suppress`'s bounds on `K` and `V` need to be tight, since the operator will actually serialize and deserialize the records (1. when it is size-constrained, 2. when it spills to disk, and 3. because this operator needs a changelog).

Reply:
I previously missed that last point... it means that Suppress requires Serdes and not just Serializers, and it needs them always, not just when it's size constrained or spilling to disk.

Reply:
Oh, and I've also just now realized that it should be called `Suppressed`, not `Suppress`, in keeping with the rest of the config objects...

This is a lot of changes. I think we'll probably have to recast votes this time.